plugin/chainconfig_manager.go

```diff
@@ -59,8 +61,19 @@ func (c *ChainConfigManager) EnableFeatures(blockHeight int64) error {
 		}
 		return err
 	}
-	for _, feature := range features {
+
+	supportBuild := c.state.GetMinBuildNumber()
+	if c.build < supportBuild {
```

Comment:
We don't need to check min build in `EnableFeature`. If the build is not supported, `chainconfig.EnableFeatures` will return error and node panics. Instead we should check the minimum build in `loadApp` by querying data from appstore https://github.com/loomnetwork/loomchain/blob/5ea160014a59315fe44de92c40a07623c1d27500/cmd/loom/loom.go#L727